test/README.md

```diff
@@ -31,7 +35,7 @@ GitHub with the `autocrlf` git config flag set to true.
 |sequential         |Yes            |Various tests that are run sequentially.|
 |testpy             |               |Test configuration utility used by various test suites.|
 |tick-processor     |No             |Tests for the V8 tick processor integration. The tests are for the logic in ```lib/internal/v8_prof_processor.js``` and  ```lib/internal/v8_prof_polyfill.js```. The tests confirm that the profile processor packages the correct set of scripts from V8 and introduces the correct platform specific logic.|
-|timers             |No             |Tests for [timing utilities](https://nodejs.org/api/timers.html) (```setTimeout``` and ```setInterval```).|
+|v8-updates         |No             |Tests for V8 performance integration.|
 
 _When a new test directory is added, make sure to update the `CI_JS_SUITES`
 variable in the `Makefile` and the `js_test_suites` variable in
```

Comment:
This bottom note can probably be removed too as it looks like both `CI_JS_SUITES` and `js_test_suites` are `default` rather than a list of test directories. This can be done in another PR if you'd rather just land this PR as-is.